db/seed.sql

```diff
@@ -7494,6 +7635,9 @@ INSERT INTO `eav_attribute` VALUES (133,4,'in_print',NULL,NULL,'int',NULL,NULL,'
 INSERT INTO `eav_attribute` VALUES (134,4,'stock_consignment',NULL,NULL,'int',NULL,NULL,'boolean','Has consignment stock?',NULL,'Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Boolean',0,1,'0',0,NULL);
 INSERT INTO `eav_attribute` VALUES (135,4,'stock_remainder',NULL,NULL,'int',NULL,NULL,'boolean','Has remainder stock?',NULL,'Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Boolean',0,1,'0',0,NULL);
 INSERT INTO `eav_attribute` VALUES (136,4,'cover',NULL,NULL,'int',NULL,NULL,'select','Cover Type',NULL,'Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Table',0,1,'4',0,NULL);
+INSERT INTO `eav_attribute` VALUES (138,4,'donation_amount',NULL,NULL,'int',NULL,NULL,'select','Donation Amount',NULL,'Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Table',0,1,'12',0,NULL);
+INSERT INTO `eav_attribute` VALUES (139,4,'image_format',NULL,NULL,'int',NULL,NULL,'select','Image Format',NULL,'Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Table',0,1,'17',0,NULL);
```

Comment:
Attribute for the drop-down list of image formats (JPEG 300dpi, JPEG 600dpi, TIFF).